canonical-websites/vanillaframework.io
# Design demo tweaks

## Done

- Capitalise Vanilla Framework link in footer
- Add correct pictograms
- Add Accessibility / Coding Standards / Browser support pages
- Make 'Getting started' content code block full width of page
- Add illustration to Guidelines row


## QA

- Pull code
- Run `./run serve --watch`
- Check site at http://0.0.0.0:8014/

Fixes #33, Fixes #34, Fixes #35, Fixes #32, Fixes #20, Fixes #21, Fixes #23
